no-std-check/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "no-std-check"
+version = "0.1.0"
+edition = "2018"
+
+[features]
+default = ["lightning/no-std", "lightning-invoice/no-std"]
+
+[dependencies]
+lightning = { version = "0.0.105", path = "../lightning", default-features = false }
```

Comment:
note that the version is optional when specifying the path.  it might be more convenient to not specify, so you don't have to update when releasing a new version.  this applies elsewhere too.

Reply:
I also wonder why the version is specified in other crates (e.g. lightning-invoice when referring to lightning), maybe worth removing in a followup PR.